src/CMakeLists.txt

```diff
@@ -69,6 +69,43 @@ add_executable(${BPFTRACE}
   ${MAIN_SRC}
 )
 
+# TODO: Add support for using "system" blazesym and linking dynamically. Then,
+#       also honor `STATIC_LINKING` properly.
+if (USE_BLAZESYM)
+  if (NOT STATIC_LINKING)
+    # It's not a strict requirement (we could just link *only* blazesym
+    # statically), but for now we enforce program-wide static linking.
+    message(FATAL_ERROR "blazesym usage requires static linking; consider setting STATIC_LINKING")
+  endif()
```

Comment:
Why is only static linking supported? Aside from the fact users generally don't have a statically linked installation (which your comments allude to), we won't get coverage in CI. I don't think it's a good idea to land without CI coverage - untested code quickly bitrots.

Reply:
Sounds good, I can remove that part.